lib/src/main/java/com/auth0/jwt/JWTCreator.java

```diff
@@ -33,6 +35,7 @@ private JWTCreator(Algorithm algorithm, Map<String, Object> headerClaims, Map<St
             SimpleModule module = new SimpleModule();
             module.addSerializer(ClaimsHolder.class, new PayloadSerializer());
             mapper.registerModule(module);
+            mapper.configure(MapperFeature.SORT_PROPERTIES_ALPHABETICALLY, true);
```

Comment:
I meant the JsonMatcher could use jackson instead of having it's own implementation of {object/list/array/map}ToString